go/gorouter/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.12.5
+
+WORKDIR /go/src/app
+COPY main.go go.mod ./
+
+ENV GO111MODULE=on
```

Comment:
is it required in **go** `1.12` ?

Reply:
I copied docker image from other go routers in this repo

Reply:
yeah, but some are outdated